static/src/javascripts/projects/common/utils/detect.js

```diff
@@ -235,7 +235,8 @@ define([
             });
         };
 
-        var woff2Candidacy = /(chrome|firefox)\/([0-9]+)/.exec(ua);
+        var excludeEdge = /^(edge\/([0-9]+))/.test(ua);
```

Comment:
looks like chrome, smells like chrome, tastes like a browser lying to my face about functional parity.